### PR TITLE
Add `ignore_nil! to documentation`

### PR DESCRIPTION
**Pull request by nambrot**
@rwz

### CODE REVIEW

**Comment by nambrot** [comment]
Any idea why this would fail?

File: README.md [hunk 1]
```diff
@@ -196,6 +196,14 @@ json.author do
 end
 ```
 
+Some languages/frameworks don't handle `null` well, in which case you can use `ignore_nil!`
```
Comment (jeremy) [inline README.md]: Specific examples? This raises more questions than it answers.

Comment (nambrot) [inline README.md]: Is this better? https://github.com/rails/jbuilder/commit/1268ef4e0763ea72a0ffb1d57a1161eecd2ef20b

**Comment by rwz** [comment]
@nambrot what if we don't mention other frameworks or languages at all? That seems secondary to the subject of the documentation. Something like this might work:

> To prevent Jbuilder from including null values in the output, you could use `ignore_nil!` method:

```ruby
json.ignore_nil!
json.foo nil
json.bar "bar"
# => { "bar": "bar" }
```

**Comment by nambrot** [comment]
Updated?

**Comment by rwz** [comment]
LGTM. Could you rebase and squash the commits into one?

**Comment by nambrot** [comment]
Did it

**Comment by rwz** [comment]
Thanks. :+1:

**Pull request merged**